packages/gitbook/src/lib/visitors.ts

```diff
@@ -128,9 +135,10 @@ export function getVisitorToken({
 export function getVisitorUnsignedClaims(args: {
     cookies: RequestCookies;
     url: URL | NextRequest['nextUrl'];
-}): Record<string, ClaimPrimitive> {
+}): { all: Record<string, ClaimPrimitive>; fromVisitorParams: Record<string, ClaimPrimitive> } {
```

Comment:
```suggestion
}): { all: Record<string, ClaimPrimitive>; params: Record<string, ClaimPrimitive> } {
```
It's better to add some comments to these properties to signify their origin